mmtrack/apis/inference.py

```diff
@@ -130,8 +130,8 @@ def inference_sot(model, image, init_bbox, frame_id):
         data['img_metas'] = data['img_metas'][0].data
 
     # forward the model
-    with torch.no_grad():
-        result = model(return_loss=False, rescale=True, **data)
+    # with torch.no_grad():
+    result = model(return_loss=False, rescale=True, **data)
```

Comment:
It will change behavior for all models.
Please change it only for prdimp models